yt_dlp/extractor/nebula.py

```diff
@@ -230,9 +259,45 @@ def _real_extract(self, url):
         return self._build_video_info(video)
 
 
-class NebulaCollectionIE(NebulaBaseIE):
-    IE_NAME = 'nebula:collection'
-    _VALID_URL = r'https?://(?:www\.)?(?:watchnebula\.com|nebula\.app)/(?!videos/)(?P<id>[-\w]+)'
+class NebulaSubscriptionsIE(NebulaBaseIE):
+    IE_NAME = 'nebula:subscriptions'
+    _VALID_URL = r'https?://(?:www\.)?(?:watchnebula\.com|nebula\.app)/myshows'
+    _TESTS = [
+        {
+            'url': 'https://nebula.app/myshows',
+            'playlist_mincount': 1,
+            'info_dict': {
+                'id': 'myshows',
+            },
+            'params': {
+                'usenetrc': True,
+            },
+        },
+    ]
+
+    CHANNEL_URL = 'https://content.watchnebula.com/library/video/?page_size=100'
+
+    def _generate_playlist_entries(self):
+        next_url = self.CHANNEL_URL
+        page_num = 1
+        while next_url:
+            channel = self._call_nebula_api(next_url, 'subscriptions', auth_type='bearer',
```

Comment:
Keep consistency. It is called `myshows` below
```suggestion
            channel = self._call_nebula_api(next_url, 'myshows', auth_type='bearer',
```